statsrelay.go

```diff
@@ -142,7 +169,7 @@ func sendPacket(buff []byte, target string, sendproto string, TCPtimeout time.Du
 			break
 		}
 		conn.Write(buff)
-		conn.Close()
+		defer conn.Close()
```

Comment:
I think we should leave the TCP connection open, re-open on errors.  That's what I've looked at doing, but I don't yet use this code path myself.  ;-)

I hesitate to do much buffering, it could definitely assist with better detection of healthy backends, but there is little we can do to prevent time skew.